### PR TITLE
responsive

### PR DESCRIPTION
**Pull request by yuvraajnarula**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,8 +51,7 @@
                         </a>
                         <div class="dropdown-menu" aria-labelledby="navbarDropdown">
                             <a class="dropdown-item" href="#mem"> Members </a>
-                            <a class="dropdown-item" href="#alumn"> Alumni</a>
-                            <a class="dropdown-item" href="#ment"> Mentors</a>
+                            <a class="dropdown-item" href="#team"> Alumni</a>
```
Comment (ion05) [inline index.html:54]: Dont change all this

**Review by ion05 — CHANGES_REQUESTED** [review]

**Comment by ion05** [comment]
Why are you changing my internal links ? @yuvraajnarula

**Comment by yuvraajnarula** [comment]
> Why are you changing my internal links ? @yuvraajnarula

coz one of them is wrong

**Comment by ion05** [comment]
> > Why are you changing my internal links ? @yuvraajnarula
> 
> coz one of them is wrong

All of them work properly and you have just removed all of them.

**Comment by ion05** [comment]
Dont mess with my links , they work fine.

**Comment by ion05** [comment]
Mere kuch changes not in this. Ek baar theek karke kar nd some dark things rehnde

**Comment by ion05** [comment]
image
fix this 
And the team section ko dark themed hi rehnde

**Comment by yuvraajnarula** [comment]
There is a problem occurring with internal linking try adding it from your end

**Comment by ion05** [comment]
Kk, dont differenitate presidents and vice presidents sepereately

**Comment by yuvraajnarula** [comment]
I think it would be better coz all mixed up doesnt look good

**Pull request merged**